packages/twenty-front/src/modules/users/contexts/UserContext.ts

```diff
@@ -1,11 +1,13 @@
 import { DateFormat } from '@/localization/constants/DateFormat';
 import { TimeFormat } from '@/localization/constants/TimeFormat';
+import { Locale } from 'date-fns';
 import { createContext } from 'react';
 
 export type UserContextType = {
   dateFormat: DateFormat;
   timeFormat: TimeFormat;
   timeZone: string;
+  dateFnsLocale: Locale;
```

Comment:
Since UserContextType  is quite generic, It feels like we might want to have our local here and not dateFnsLocale? Maybe convert later downstream? There will probably be other use-case that would leverage this locale